src/index.html

```diff
@@ -9,10 +9,63 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <ul class="stars__list">
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+      </ul>
+    </div>
+    <div class="stars stars--1">
+      <ul class="stars__list">
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+      </ul>
+    </div>
+    <div class="stars stars--2">
+      <ul class="stars__list">
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+      </ul>
+    </div>
+    <div class="stars stars--3">
+      <ul class="stars__list">
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+      </ul>
+    </div>
+    <div class="stars stars--4">
+      <ul class="stars__list">
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+      </ul>
+    </div>
+    <div class="stars stars--5">
+      <ul class="stars__list">
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+        <li class="stars__list-item"></li>
+      </ul>
+    </div>
```

Comment:
```suggestion
    <div class="stars stars--0">
      <ul class="stars__list">
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
      </ul>
    </div>
    
    <div class="stars stars--1">
      <ul class="stars__list">
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
      </ul>
    </div>
    
    <div class="stars stars--2">
      <ul class="stars__list">
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
      </ul>
    </div>
    
    <div class="stars stars--3">
      <ul class="stars__list">
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
      </ul>
    </div>
    
    <div class="stars stars--4">
      <ul class="stars__list">
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
      </ul>
    </div>
    
    <div class="stars stars--5">
      <ul class="stars__list">
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
        <li class="stars__list-item"></li>
      </ul>
    </div>
```

Reply:
But also you don't really need `div` here. You can delete `div` and add `stars stars--n` styles to `ul`